SQFT/install.sh

```diff
@@ -21,7 +21,7 @@ pip install git+https://github.com/AutoGPTQ/AutoGPTQ@866b4c8
 
 # nncf
 git clone https://github.com/openvinotoolkit/nncf.git
-cd nncf && git checkout v2.12.0 && git apply --ignore-space-change --ignore-whitespace ${SQFT_PATH}/patches/nncf-v2.12.0.patch && pip install -e . && cd ..
+cd nncf && git checkout f143e1c && git apply --ignore-space-change --ignore-whitespace ${SQFT_PATH}/patches/nncf-f143e1c.patch && pip install -e . && cd ..
```

Comment:
Do you have to be tied to a commit and not a full release?

Reply:
Yes, we are using latest features in the development branch. Thanks!